python-package/securexgboost/core.py

```diff
@@ -370,7 +370,7 @@ class DMatrix(object):
     def __init__(self, data, encrypted=False, label=None, missing=None,
                  weight=None, silent=False,
                  feature_names=None, feature_types=None,
-                 nthread=None):
+                 nthread=None, username = None):
```

Comment:
Please change this to `username=None`, i.e. without the spaces

Reply:
done